cobalt/network/network_module.h

```diff
@@ -59,6 +59,8 @@ enum ClientHintHeadersCallType : int32_t {
   kCallTypeXHR = (1u << 5),
 };
 
+const char kQuicEnabledPersistentSettingsKey[] = "QUIC";
```

Comment:
This is migrating an old / legacy setting though - which is already in client codebase. We wanted this to just fit in, so that Cobalt 24 actually properly enables/disables this. We also don't want old/new behavior side by side here, as the old behavior already has a very large control group.

Reply:
Fair enough there, yes. Something to be said for keeping those mapped 1:1 but i don't think it matters that much.

Reply:
done